std_msgs/Byte number
std_msgs/Byte condition
std_msgs/Byte environment

---
